qcelemental/tests/test_molparse_from_string.py

```diff
@@ -0,0 +1,1136 @@
+import sys
+import copy
+#import pprint
+
+import pytest
+import numpy as np
+
+from utils_compare import *
+#from addons import *
+
+import qcelemental
+
+#import qcdb
+
+subject1 = """O 0 0   0
+no_com
+
+H 1 ,, 0 \t  0 # stuff-n-nonsense"""
+
+ans1 = {
+    'geom': [0., 0., 0., 1., 0., 0.],
+    'elbl': ['O', 'H'],
+    'fix_com': True,
+    'fragment_separators': [],
+    'fragment_charges': [None],
+    'fragment_multiplicities': [None],
+    'fragment_files': [],
+    'geom_hints': [],
+    'hint_types': [],
+}
+
+fullans1a = {
+    'geom': np.array([0., 0., 0., 1., 0., 0.]),
+    'elea': np.array([16, 1]),
+    'elez': np.array([8, 1]),
+    'elem': np.array(['O', 'H']),
+    'mass': np.array([15.99491462, 1.00782503]),
+    'real': np.array([True, True]),
+    'elbl': np.array(['', '']),
```

Comment:
We should probably have a name for these non-schema objects. It is ok to carry around extra metadata, but the fact that units can be something besides Bohr is a dealbreaker.